doc/integrator/checker.rst

```diff
@@ -150,6 +151,17 @@ using the ``routes`` configuration as an array of route names to check::
   * ``params`` the used query string as a dictionary.
   * ``level``
 
+Infrastructure
+--------------
+
+We can use the following configurations:
```

Comment:
Suggestion:
To be able to verify that your application is operational, you can define a "checker". Configure the checker as follows:

Reply:
=> In the ``checker`` configuration you can add the following configuration to solve some connection issues.

Looks good?

Reply:
How about:

If you experience connection issues with your ``checker``, the following configuration options may be useful for you:

Reply:
+1, Thanks :-)